### ISSUE TITLE
RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by DLevai94** [benmosher/eslint-plugin-import#1260]
```
export * from '.';
```
Causes error.
```
RangeError: Maximum call stack size exceeded
    at Function.keys (<anonymous>)
    at hashObject (/Users/davidlevai/Desktop/Work/Classy/Projects/BES/bybesapp/node_modules/eslint-module-utils/hash.js:46:10)
    at hashify (/Users/davidlevai/Desktop/Work/Classy/Projects/BES/bybesapp/node_modules/eslint-module-utils/hash.js:18:5)
    at Object.keys.sort.forEach.key (/Users/davidlevai/Desktop/Work/Classy/Projects/BES/bybesapp/node_modules/eslint-module-utils/hash.js:49:5)
    at Array.forEach (<anonymous>)
    at hashObject (/Users/davidlevai/Desktop/Work/Classy/Projects/BES/bybesapp/node_modules/eslint-module-utils/hash.js:46:30)
    at hashify (/Users/davidlevai/Desktop/Work/Classy/Projects/BES/bybesapp/node_modules/eslint-module-utils/hash.js:18:5)
    at Object.keys.sort.forEach.key (/Users/davidlevai/Desktop/Work/Classy/Projects/BES/bybesapp/node_modules/eslint-module-utils/hash.js:49:5)
    at Array.forEach (<anonymous>)
    at hashObject (/Users/davidlevai/Desktop/Work/Classy/Projects/BES/bybesapp/node_modules/eslint-module-utils/hash.js:46:30)
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

https://github.com/eslint/eslint/issues/11274#issuecomment-454557280

**Comment by ljharb** [comment]
Certainly this shouldn't crash; however, that particular export statement is utterly nonsensical :-)

**Comment by DLevai94** [comment]
@ljharb thanks for the quick answer. Yeah, it could be. @dobosmarton could you pls explain why we needed this file? `src/components/Pager/index.js`

**Comment by seyfer** [comment]
was it fixed and tagged in a new version already?

**Comment by ljharb** [comment]
Looks like it was fixed in #1496, and released in v2.19.0
